packaging/tags.py

```diff
@@ -410,7 +410,7 @@ def _mac_binary_formats(version, cpu_arch):
     if cpu_arch in {"arm64", "x86_64"}:
         formats.append("universal2")
 
-    if cpu_arch in {"x86_64", "i386", "ppc64", "ppc"}:
+    if cpu_arch in {"x86_64", "i386", "ppc64", "ppc", "intel"}:
```

Comment:
Why do we add only "universal" and not "universal2" for "intel"? (I'm not sure when this arch name is used)

Reply:
See https://github.com/python/cpython/blob/41761933c1c30bb6003b65eef1ba23a83db4eae4/Mac/README.rst#21-flavors-of-universal-binaries

`intel` refers to a universal binary that works on `i386` and `x86_64` -- which are both supported by `universal`.